snuba/utils/metrics/backends/dummy.py

```diff
@@ -0,0 +1,43 @@
+from collections import Mapping
+from typing import Optional, Union
+
+from snuba.utils.metrics import Tags
+from snuba.utils.metrics.backends.abstract import MetricsBackend
+
+
+class DummyMetricsBackend(MetricsBackend):
```

Comment:
It probably would be good to have a test that exercises all backends. Only the timing is covered here (without tag validation.)